.jenkins/pipelines/scheduled-tests.Jenkinsfile

```diff
@@ -0,0 +1,189 @@
+/* A Jenkins pipeline that will handle code coverage and nightly tests
+*  These are the original pipelines:
+*  https://github.com/deislabs/mystikos/blob/main/.azure_pipelines/ci-pipeline-code-coverage-nightly.yml
+*/
+
+pipeline {
+    agent {
+        label 'ACC-1804-DC4'
+    }
+    options {
+        timeout(time: 600, unit: 'MINUTES')
+    }
+    parameters {
+        string(name: "REPOSITORY", defaultValue: "deislabs")
+        string(name: "BRANCH", defaultValue: "main", description: "Branch to build")
+        choice(name: "TEST_CONFIG", choices:['Nightly', 'Code Coverage'], description: "Test configuration to execute")
+    }
+    environment {
+        MYST_SCRIPTS =    "${WORKSPACE}/scripts"
+        JENKINS_SCRIPTS = "${WORKSPACE}/.jenkins/scripts"
+        BUILD_USER = sh(
+            returnStdout: true,
+            script: 'echo \${USER}'
+        )
+        MYST_NIGHTLY_TEST = 1
+        MYST_ENABLE_GCOV = 1
```

Comment:
This would be set as well when the nightly pipeline is ran, yet the original nightly pipeline does not set this. What are the implications? If there are any concerns it would be better to set this only when running the Code Coverage pipeline.

Reply:
This file has the possibility of replacing the nightly pipeline; I think we should attempt that outside of this PR. I'll investigate the differences when/if we end up replacing the nightly pipeline with this file.